src/simulator/views/map/map_view.py

```diff
@@ -225,7 +241,7 @@ def eager_refresh():
             for p in np.ndindex(self.map.traversables_data.shape):
                 self.__cube_modified[p] = self.map.traversables_data[p]
         
-        clr = self._services.state.effective_view.colours[MapData.TRAVERSABLES]()
+        clr = TRANSPARENT if self.__is_ogm else self._services.state.effective_view.colours[MapData.TRAVERSABLES]()
```

Comment:
are we no longer able to change the color of the traversables in an ogm?

Reply:
ie this looks like the color of traversables is just hard set to transparent? maybe im wrong though

Reply:
Fair point actually, we should probably still be able to see the base colour `traversables` and people can then toggle that to invisible if they like right?

Reply:
yeah i dont think it should be a special case; only the obstacles should change in an ogm imo. perhaps we can set the default of all the other 3d map views of traversables to transparent as well btw?

Reply:
cuz i agree its hard to see the obstacles in an ogm if the traversables arent transparent; but its also the case for the 3d maps as well

Reply:
Changed to have no special treatment.